src/app/export-excel/excel-export-sample-1/excel-export-sample-1.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, ViewChild } from "@angular/core";
+
+import { IgxExcelExporterService,
+         IgxExcelExporterOptions,
+         IColumnExportingEventArgs,
+         IRowExportingEventArgs,
+         IExcelExportEndedEventArgs } from "igniteui-angular/services/index";
+
+import { IgxGridComponent } from "igniteui-angular/grid/grid.component";
+
+@Component({
+  selector: "app-excel-export-sample-1",
+  templateUrl: "./excel-export-sample-1.component.html",
+  styleUrls: ["./excel-export-sample-1.component.scss"]
+})
+export class ExcelExportSample1Component {
+
+  @ViewChild("igxGrid1") igxGrid1: IgxGridComponent;
+  localData = [
+    { Name: "Eric Ridley", Age: "26" },
+    { Name: "Alanis Brook", Age: "22" },
+    { Name: "Jonathan Morris", Age: "23" }
+  ];
+
+  constructor() {
+  }
+
+  exportButtonHandler() {
+    let ees: IgxExcelExporterService = new IgxExcelExporterService();
```

Comment:
@rmkrmk: Don't create services directly. Instead add them to the providers array of the module/component and inject them in the constructor. The angular injector will handle the creation for you.